nntrainer/tensor/blas_neon.cpp

```diff
@@ -248,7 +248,7 @@ void sgemv_transpose_neon(const float *A, const float *X, float *Y,
     unsigned int n = cols / 4;
     bool *initialized = (bool *)malloc(sizeof(bool) * n);
     if (initialized == nullptr) {
-      ml_loge("failed to malloc");
+      ml_loge("Error : Memory allocation failed");
```

Comment:
initialized is not transfered to its caller.

Consider using ```auto initialized = make_unique<bool []>(n);```. You won't need nullptr check, either, then.

Reply:
Thank you for review, i will fix and update